README.md

```diff
@@ -2038,6 +2038,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Waqe](https://github.com/waq-e)
   - [Wayne O'Mahony](https://github.com/wayneomahony)
   - [WebDevWill](https://github.com/WebDevWill)
+  - [Will Maple](https://github.com/willlytton)
```

Comment:
Your name should be after Wilgerth Mouton.
If you need any help with updating, let us know. 🙂